src/styles/blocks/header.scss

```diff
@@ -0,0 +1,57 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  .header__logo {
+    height: 40px;
+  }
+}
+
+.nav__list {
```

Comment:
according to the current naming ".nav" is a separate block and have to be in separate file